docs/source/Publications.rst

```diff
@@ -0,0 +1,12 @@
+Publications
+============
+
+*QMAP* is academic software. Thus, many of its built-in algorithms have been published as scientific papers.
+
+If you use *QMAP* in your work, we would appreciate if you cited :cite:labelpar:`zulehnerEfficientMethodologyMapping2019` when using the heuristic mapper and :cite:labelpar:`willeMappingQuantumCircuits2019` when using the exact mapper.
+Furthermore, if you use any of the particular algorithms such as
+- the heuristic mapping scheme using teleportation :cite:labelpar:`hillmichExlpoitingQuantumTeleportation2021`
```

Comment:
you need to introduce an empty line before and after the itemize in order for it to work.

Reply:
Got it.